src/components/Tooltip/index.jsx

```diff
@@ -1,58 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import FDS from 'lib/dictionary/js/styleConstants';
-import Popover, { VALID_POSITIONS } from 'components/Popover';
-
-export const DEFAULT_WIDTH = '240px';
-export const DELAY_MS = 350;
-export const DEFAULT_POSITION = 'bottom';
-const DEFAULT_TEXT_ALIGN = 'center';
-export const VALID_TEXT_ALIGN = ['left', 'center', 'right'];
+import Tippy from '@tippyjs/react';
 
 const Tooltip = ({
   trigger,
   message,
-  maxWidth,
-  position = DEFAULT_POSITION,
-  textAlign = DEFAULT_TEXT_ALIGN,
+  position = 'bottom',
+  maxWidth = 240,
+  textAlign = 'center',
 }) => (
-  <Popover
-    interactionMode="hover"
-    trigger={<span style={{ cursor: 'help' }}>{trigger}</span>}
-    distance={parseInt(FDS.SPACE_HALF, 10)}
-    position={position}
-    alignment="center"
-    transitionName="GrowFast"
-    delay={DELAY_MS}
+  <Tippy
+    arrow={false}
+    theme="tooltip"
+    delay={[350, 0]}
+    maxWidth={maxWidth}
+    content={message}
+    placement={position}
+    zIndex={FDS.ZINDEX_MODAL}
+    moveTransition="transform 0.2s ease-out"
+    appendTo={document.body}
+    className={`bgColor--charcoal inverted align--${textAlign} padding--top--half padding--bottom--half padding--left padding--right elevation--2 rounded--all fontSize--s fontWeight--bold`}
   >
-    <div
-      role="tooltip"
-      style={{ maxWidth: maxWidth ? `${maxWidth}px` : DEFAULT_WIDTH }}
-      className={`bgColor--charcoal inverted align--${textAlign} padding--top--half padding--bottom--half padding--left padding--right elevation--2 rounded--all fontSize--s fontWeight--bold`}
-    >
-      {message}
-    </div>
-  </Popover>
+    {trigger}
+  </Tippy>
 );
 
 Tooltip.propTypes = {
   /** JSX - Acts as a positioning reference for the popover and triggers active state */
-  trigger: PropTypes.oneOfType([PropTypes.node, PropTypes.element]).isRequired,
+  trigger: PropTypes.oneOfType([PropTypes.node, PropTypes.element]),
 
   /** Content of the tooltip */
   message: PropTypes.string.isRequired,
 
   /** Maximum width of tooltip */
   maxWidth: PropTypes.number,
 
-  /**
-   * Position preference of tooltip.
-   * `top` for example, will place the tooltip above the trigger.
-   */
-  position: PropTypes.oneOf(VALID_POSITIONS),
-
+  /** Position preference of tooltip. */
+  position: PropTypes.oneOf([
```

Comment:
At first I wondered if we should only include centered positioning options (left, right, bottom, left), but I can't think of a good reason not to include all positioning options.

We might want to put this array somewhere shared for Popover, but we can decide on that as you work with the Popover conversion

Reply:
I mean less options is better if we know they don't need them. That being said, that would be easy to go through and do a breaking change at a later date, if we discover what the "UX" is from our designers or if we see common patterns.

Reply:
I'm good with leaving all these in for now, agree with your point